sklearn/model_selection/tests/test_validation.py

```diff
@@ -857,13 +855,12 @@ def split(self, X, y=None, groups=None):
 
     X, y = load_iris(return_X_y=True)
 
-    warning_message = ('Number of classes in training fold (2) does '
-                       'not match total number of classes (3). '
+    warning_message = (r'Number of classes in training fold \(2\) does '
+                       'not match total number of classes \(3\). '
```

Comment:
```suggestion
                       r'not match total number of classes \(3\). '
```

Reply:
@jeremiedbb 

Thanks for the suggestion code!
I applied this.